Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -702,70 +702,77 @@ static RefPtr<CSSValue> consumeFontVariantEastAsian(CSSParserTokenRange& range)
     if (range.peek().id() == CSSValueNormal)
         return consumeIdent(range);
     
-    RefPtr<CSSValueList> values = CSSValueList::createSpaceSeparated();
     std::optional<FontVariantEastAsianVariant> variant;
     std::optional<FontVariantEastAsianWidth> width;
     std::optional<FontVariantEastAsianRuby> ruby;
     
-    while (!range.atEnd()) {
-        if (range.peek().type() != IdentToken)
-            return nullptr;
+    auto parseSomethingWithoutError = [&range, &variant, &width, &ruby] () {
+        bool hasParsedSomething = false;
 
-        auto id = range.peek().id();
+        for (;;) {
```

Comment:
```suggestion
        while (true) {
```

nit: `while (true)` is much more largely used in WebKit: https://searchfox.org/wubkat/search?q=while+%28true%29&path=.cpp&case=false&regexp=false

compared to `for (;;)`: https://searchfox.org/wubkat/search?q=for+%28%3B%3B%29+%7B&path=cpp&case=false&regexp=false

Reply:
Changed to `while (true)`. The repartition of usage in our codebase (1/3 vs 2/3) is not great, we should agree on one of them and add a style checker rule for it.